src/core_modules/capture-core/components/WidgetEventEdit/EditEventDataEntry/EditEventDataEntry.component.js

```diff
@@ -189,12 +189,12 @@ const buildScheduleDateSettingsFn = () => {
             calendarWidth: 350,
             label: props.formFoundation.getLabel('scheduledAt'),
             disabled: true,
+            required: false,
```

Comment:
Currently, the "Enter details now" case of the "Related Stages" feature assumes that the event-to-linked-from always has a scheduledAt date. Perhaps we could use the `occurredAt` date if the `scheduledAt` is empty to generate the linked event payload? WDYT?
The file to look into is [getConvertedRelatedStageEvent](https://github.com/dhis2/capture-app/blob/master/src/core_modules/capture-core/components/DataEntries/converters/getConvertedRelatedStageEvent/getConvertedRelatedStageEvent.js#L71).

Reply:
Yes, I realised there were more cases to handle here (hence, why I converted it to a draft PR). I think every workflow is handled now, including the one you mentioned here (I tested all the workflows I could think of).